python/list_and_match_messages.py

```diff
@@ -33,22 +40,29 @@ def parse_arguments():
     """
         Step 2 : Send an api request to get list of messages based on filters
     """
-    params = {'page': APIConstants.DEFAULT_BULK_CREATE_LIMIT,
-              'fields': (CommonConstants.PHONE_NUMBER_FIELD, CommonConstants.ID_FIELD, CommonConstants.SEND_ON_FIELD),
-              "filters": dict(list(map(str.strip, s.split('='))) for s in args.params)}
+    params = {'fields': ','.join(
+        [CommonConstants.PHONE_NUMBER_FIELD, CommonConstants.ID_FIELD, CommonConstants.SEND_ON_FIELD]),
+    }
+    filters = dict(list(map(str.strip, s.split('='))) for s in args.params)
```

Comment:
@ashwini-ad If we are splitting params with "=" sign then it will work when all params will have "=" sign only.

Based on your description of params argument, You maintain that >= and <= will also allow, So in >= and <= cases how this split will work. I feel it will generate the wrong filters directory.

Reply:
We would not use signs here for greater than, but use "__gt" and "__lt" like we do in django queries

Reply:
@ashwini-ad in that case please change the example you are providing the same in above of the method. Make it consistance.